pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# What to add
+
+The title and link to a podcast that belongs in our list.
+
+The current categories are:
+- Javascript
+- General Technology & Web Development 
+- Other Technology-Specific 
+- Women in Tech
+
+# How to Contribute
```

Comment:
You should not repeat the how to contribute guide on a PR template, please remove that section